doc/renaming_table.xml

```diff
@@ -0,0 +1,29 @@
+<Table Align="|l|l|">
+<Caption> Renaming</Caption>
+<HorLine/>
+<Row>
+	<Item> <E>Old Name</E> </Item>
+	<Item> <E>New Name</E> </Item>
+</Row>
+<HorLine/>
+<Row>
+	<Item> RecognitionInfoFamily </Item>
```

Comment:
```suggestion
	<Item> <C>RecognitionInfoFamily</C> </Item>
```
and so on.

Also, perhaps index keys could be added for `RecognitionInfoFamily`, etc, so that the old names still appear in the index (but they now direct you to the correct place now, so `RecognitionInfoFamily` would take you to the documentation for `RecogNodeFamily`, etc). I _think_ this is popular with GAP Doc, but I can't remember how.

Reply:
Perhaps you have to add `<Index><C>RecognitionInfoFamily</C></Index>` to the documentation for `RecogNodeFamily`.

Reply:
That works for me. Nice, thanks! Adding the old names to the index sounds like a good idea to me.

I think I would prefer it to have the index entries of the old names link to the renaming chapter instead. That way there can't be any confusion about why one clicks e.g. on `RecognitionInfoFamily`, but does not find any documentation for `RecognitionInfoFamily`. Especially since sometimes a new name @FriedrichRober and I came up with can be very different from the old name, if the latter was very cryptic.

Reply:
Yeah that's a better idea.

Reply:
In order for such an entry to be sorted correctly into the index one also needs to specify a sort `Key` like this:
`<Index Key="RecognitionInfoFamily"><C>RecognitionInfoFamily</C></Index>`